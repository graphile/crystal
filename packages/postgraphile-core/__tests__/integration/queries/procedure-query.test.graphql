query {
  jsonIdentity(json: "{\"a\":1,\"b\":2,\"c\":3}")
  jsonbIdentity(json: "{\"a\":1,\"b\":2,\"c\":3}")
  jsonIdentityIssue85: jsonIdentity(json:  "[{\"amount\":\"44\"},{\"amount\":null}]")
  jsonbIdentityIssue85: jsonbIdentity(json: "[{\"amount\":\"44\"},{\"amount\":null}]")
  add1Query(arg0: 1, arg1: 2)
  add2Query(a: 2, b: 2)
  add3Query(arg1: 5)
  add4Query(arg0: 1, b: 3)
  optionalMissingMiddle1(arg0: 1, c: 7) # expect 10 (b defaults to 2)
  optionalMissingMiddle1_2: optionalMissingMiddle1(arg0: 1, b: 8, c: 7) # expect 16
  optionalMissingMiddle2(a: 1, c: 7) # expect 10 (b defaults to 2)
  optionalMissingMiddle3(a: 1, c: 7) # expect 10 because we can specify $1 positionally and $3 via name, and b defaults to 2
  optionalMissingMiddle4(arg0: 1, arg2: 7) # expect null because we can't specify $3 without specifying $2 (= null)
  optionalMissingMiddle5(a: 1, arg2: 7) # expect null because we can't specify $3 without specifying $2 (= null)
  typesQuery(a: "50", b: false, c: "xyz", d: [1, 2, 3], e: "{\"a\":1,\"b\":2,\"c\":3}", f: { start: { value: 1, inclusive: false }, end: { value: 5, inclusive: false } })
  empty: typesQuery(a: "50", b: false, c: "", d: [], e: "{}", f: {})
  compoundTypeQuery(object: { a: 419, b: "easy cheesy baked potatoes", c: RED, e: BAR_FOO, f: _EMPTY_, g: {seconds: 12, minutes: 3, hours: 2}, fooBar: 8 }) { a b c d e f g { hours minutes seconds } fooBar }
  compoundTypeSetQuery(first: 5) { edges { cursor node { a b c d e f f g { hours minutes seconds } fooBar } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  compoundTypeArrayQuery(object: { a: 419, b: "easy cheesy baked potatoes", c: RED, e: BAR_FOO, f: _EMPTY_, g: {seconds: 12, minutes: 3, hours: 2}, fooBar: 8 }) { a b c d e f f g { hours minutes seconds } fooBar }
  tableQuery(id: 5) { nodeId id headline authorId }
  tableSetQuery { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQuery_sorted: tableSetQuery(orderBy: NAME_ASC) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQuery_condition: tableSetQuery(condition: {name: "Budd Deey"}) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryWithOffset: tableSetQuery(before: "WyJuYXR1cmFsIiw1XQ==", after: "WyJuYXR1cmFsIiwzXQ==") { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryWithOffset2: tableSetQuery(after: "WyJuYXR1cmFsIiwzXQ==", before: "WyJuYXR1cmFsIiw1XQ==") { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryWithOffset3: tableSetQuery(before: "WyJuYXR1cmFsIiw1XQ==", last: 2) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryWithOffset4: tableSetQuery(before: "WyJuYXR1cmFsIiw1XQ==", first: 2) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryWithOffset5: tableSetQuery(after: "WyJuYXR1cmFsIiwzXQ==", first: 2) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryWithOffset6: tableSetQuery(first: 2, offset: 2) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  # There's only 6 people, so there should be no next page
  tableSetQueryWithOffset7: tableSetQuery(first: 2, offset: 4) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  # There should be no previous page
  tableSetQueryWithOffset8: tableSetQuery(first: 2, offset: 0) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  # There should be no previous or next page
  tableSetQueryWithOffset9: tableSetQuery(first: 6, offset: 0) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  # Requesting before the first record returns no results, and thus has hasNextPage = false and hasPreviousPage = false as there's no cursors to use for pagination
  tableSetQueryWithOffset10: tableSetQuery(before: "WyJuYXR1cmFsIiwxXQ==", last: 2) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  # Using after and offset together should not throw
  tableSetQueryWithOffset11: tableSetQuery(after: "WyJuYXR1cmFsIiwxXQ==", offset: 1, first: 2) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryPlpgsql(first:2) { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  tableSetQueryPlpgsqlOffset2: tableSetQueryPlpgsql(first:2, after: "WyJuYXR1cmFsIiwyXQ==") { edges { cursor node { name } } pageInfo { startCursor endCursor hasNextPage hasPreviousPage } }
  intSetQuery(x: 5, z: 6) { edges { cursor node } totalCount }
  noArgsQuery
  staticBigInteger { edges { node } totalCount }

  queryCompoundTypeArray(object: { a: 419, b: "easy cheesy baked potatoes", c: RED, e: BAR_FOO, f: _EMPTY_, g: {hours: 5 }, fooBar: 8 }) { a b c d e f g { seconds minutes hours } fooBar }
  queryTextArray
  queryIntervalArray { seconds minutes hours days months years }
  queryIntervalSet { nodes { seconds minutes hours days months years } edges { node { seconds minutes hours } cursor } totalCount }
}
