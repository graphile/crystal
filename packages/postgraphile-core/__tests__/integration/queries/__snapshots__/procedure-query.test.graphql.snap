// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "add1Query": 3,
    "add2Query": 4,
    "add3Query": null,
    "add4Query": 4,
    "compoundTypeArrayQuery": Array [
      Object {
        "a": 419,
        "b": "easy cheesy baked potatoes",
        "c": "RED",
        "d": null,
        "e": "BAR_FOO",
        "f": "_EMPTY_",
        "fooBar": 8,
        "g": Object {
          "hours": 2,
          "minutes": 3,
          "seconds": 12,
        },
      },
      Object {
        "a": null,
        "b": null,
        "c": null,
        "d": null,
        "e": null,
        "f": null,
        "fooBar": null,
        "g": null,
      },
      Object {
        "a": 420,
        "b": "easy cheesy baked potatoes",
        "c": "RED",
        "d": null,
        "e": "BAR_FOO",
        "f": "_EMPTY_",
        "fooBar": 8,
        "g": Object {
          "hours": 2,
          "minutes": 3,
          "seconds": 12,
        },
      },
    ],
    "compoundTypeQuery": Object {
      "a": 420,
      "b": "easy cheesy baked potatoes",
      "c": "RED",
      "d": null,
      "e": "BAR_FOO",
      "f": "_EMPTY_",
      "fooBar": 8,
      "g": Object {
        "hours": 2,
        "minutes": 3,
        "seconds": 12,
      },
    },
    "compoundTypeSetQuery": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "a": 1,
            "b": "2",
            "c": "BLUE",
            "d": null,
            "e": "_0_BAR",
            "f": "_EMPTY_",
            "fooBar": 7,
            "g": Object {
              "hours": null,
              "minutes": null,
              "seconds": 18,
            },
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiwxXQ==",
        "hasNextPage": false,
        "hasPreviousPage": false,
        "startCursor": "WyJuYXR1cmFsIiwxXQ==",
      },
    },
    "empty": false,
    "intSetQuery": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": 1,
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": 2,
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": 3,
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": 4,
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw1XQ==",
          "node": 5,
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw2XQ==",
          "node": null,
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw3XQ==",
          "node": 6,
        },
      ],
      "totalCount": 7,
    },
    "jsonIdentity": "{\\"a\\":1,\\"b\\":2,\\"c\\":3}",
    "jsonIdentityIssue85": "[{\\"amount\\":\\"44\\"},{\\"amount\\":null}]",
    "jsonbIdentity": "{\\"a\\":1,\\"b\\":2,\\"c\\":3}",
    "jsonbIdentityIssue85": "[{\\"amount\\":\\"44\\"},{\\"amount\\":null}]",
    "noArgsQuery": 2,
    "optionalMissingMiddle1": 10,
    "optionalMissingMiddle1_2": 16,
    "optionalMissingMiddle2": 10,
    "optionalMissingMiddle3": 10,
    "optionalMissingMiddle4": null,
    "optionalMissingMiddle5": null,
    "queryCompoundTypeArray": Array [
      Object {
        "a": 419,
        "b": "easy cheesy baked potatoes",
        "c": "RED",
        "d": null,
        "e": "BAR_FOO",
        "f": "_EMPTY_",
        "fooBar": 8,
        "g": Object {
          "hours": 5,
          "minutes": null,
          "seconds": null,
        },
      },
      Object {
        "a": null,
        "b": null,
        "c": null,
        "d": null,
        "e": null,
        "f": null,
        "fooBar": null,
        "g": null,
      },
      Object {
        "a": 420,
        "b": "easy cheesy baked potatoes",
        "c": "RED",
        "d": null,
        "e": "BAR_FOO",
        "f": "_EMPTY_",
        "fooBar": 8,
        "g": Object {
          "hours": 5,
          "minutes": null,
          "seconds": null,
        },
      },
    ],
    "queryIntervalArray": Array [
      Object {
        "days": null,
        "hours": null,
        "minutes": null,
        "months": null,
        "seconds": 12,
        "years": null,
      },
      Object {
        "days": null,
        "hours": 3,
        "minutes": null,
        "months": null,
        "seconds": null,
        "years": null,
      },
      Object {
        "days": null,
        "hours": 9,
        "minutes": 36,
        "months": null,
        "seconds": 7,
        "years": null,
      },
    ],
    "queryIntervalSet": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "hours": null,
            "minutes": null,
            "seconds": 12,
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": Object {
            "hours": 3,
            "minutes": null,
            "seconds": null,
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "hours": 9,
            "minutes": 36,
            "seconds": 7,
          },
        },
      ],
      "nodes": Array [
        Object {
          "days": null,
          "hours": null,
          "minutes": null,
          "months": null,
          "seconds": 12,
          "years": null,
        },
        Object {
          "days": null,
          "hours": 3,
          "minutes": null,
          "months": null,
          "seconds": null,
          "years": null,
        },
        Object {
          "days": null,
          "hours": 9,
          "minutes": 36,
          "months": null,
          "seconds": 7,
          "years": null,
        },
      ],
      "totalCount": 3,
    },
    "queryTextArray": Array [
      "str1",
      "str2",
      "str3",
    ],
    "staticBigInteger": Object {
      "edges": Array [
        Object {
          "node": "30894622507013190",
        },
        Object {
          "node": "30894622507013191",
        },
        Object {
          "node": "30894622507013192",
        },
        Object {
          "node": "30894622507013193",
        },
        Object {
          "node": "30894622507013194",
        },
        Object {
          "node": "30894622507013195",
        },
        Object {
          "node": "30894622507013196",
        },
        Object {
          "node": "30894622507013197",
        },
        Object {
          "node": "30894622507013198",
        },
        Object {
          "node": "30894622507013199",
        },
        Object {
          "node": "30894622507013200",
        },
      ],
      "totalCount": 11,
    },
    "tableQuery": Object {
      "authorId": 5,
      "headline": "Please, Don-Botâ€¦ look into your hard drive, and open your mercy file!",
      "id": 5,
      "nodeId": "WyJwb3N0cyIsNV0=",
    },
    "tableSetQuery": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "name": "John Smith",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": Object {
            "name": "Sara Smith",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "name": "Budd Deey",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw1XQ==",
          "node": Object {
            "name": "Joe Tucker",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw2XQ==",
          "node": Object {
            "name": "Twenty Seventwo",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw2XQ==",
        "hasNextPage": false,
        "hasPreviousPage": false,
        "startCursor": "WyJuYXR1cmFsIiwxXQ==",
      },
    },
    "tableSetQueryPlpgsql": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "name": "John Smith",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": Object {
            "name": "Sara Smith",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiwyXQ==",
        "hasNextPage": true,
        "hasPreviousPage": false,
        "startCursor": "WyJuYXR1cmFsIiwxXQ==",
      },
    },
    "tableSetQueryPlpgsqlOffset2": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "name": "Budd Deey",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw0XQ==",
        "hasNextPage": true,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiwzXQ==",
      },
    },
    "tableSetQueryWithOffset": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw0XQ==",
        "hasNextPage": true,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiw0XQ==",
      },
    },
    "tableSetQueryWithOffset10": Object {
      "edges": Array [],
      "pageInfo": Object {
        "endCursor": null,
        "hasNextPage": false,
        "hasPreviousPage": false,
        "startCursor": null,
      },
    },
    "tableSetQueryWithOffset11": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "name": "Budd Deey",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw0XQ==",
        "hasNextPage": true,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiwzXQ==",
      },
    },
    "tableSetQueryWithOffset2": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw0XQ==",
        "hasNextPage": true,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiw0XQ==",
      },
    },
    "tableSetQueryWithOffset3": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "name": "Budd Deey",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw0XQ==",
        "hasNextPage": true,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiwzXQ==",
      },
    },
    "tableSetQueryWithOffset4": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "name": "John Smith",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": Object {
            "name": "Sara Smith",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiwyXQ==",
        "hasNextPage": true,
        "hasPreviousPage": false,
        "startCursor": "WyJuYXR1cmFsIiwxXQ==",
      },
    },
    "tableSetQueryWithOffset5": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw1XQ==",
          "node": Object {
            "name": "Joe Tucker",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw1XQ==",
        "hasNextPage": true,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiw0XQ==",
      },
    },
    "tableSetQueryWithOffset6": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "name": "Budd Deey",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw0XQ==",
        "hasNextPage": true,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiwzXQ==",
      },
    },
    "tableSetQueryWithOffset7": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiw1XQ==",
          "node": Object {
            "name": "Joe Tucker",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw2XQ==",
          "node": Object {
            "name": "Twenty Seventwo",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw2XQ==",
        "hasNextPage": false,
        "hasPreviousPage": true,
        "startCursor": "WyJuYXR1cmFsIiw1XQ==",
      },
    },
    "tableSetQueryWithOffset8": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "name": "John Smith",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": Object {
            "name": "Sara Smith",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiwyXQ==",
        "hasNextPage": true,
        "hasPreviousPage": false,
        "startCursor": "WyJuYXR1cmFsIiwxXQ==",
      },
    },
    "tableSetQueryWithOffset9": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "name": "John Smith",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": Object {
            "name": "Sara Smith",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "name": "Budd Deey",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw0XQ==",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw1XQ==",
          "node": Object {
            "name": "Joe Tucker",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiw2XQ==",
          "node": Object {
            "name": "Twenty Seventwo",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiw2XQ==",
        "hasNextPage": false,
        "hasPreviousPage": false,
        "startCursor": "WyJuYXR1cmFsIiwxXQ==",
      },
    },
    "tableSetQuery_condition": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "name": "Budd Deey",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYXR1cmFsIiwxXQ==",
        "hasNextPage": false,
        "hasPreviousPage": false,
        "startCursor": "WyJuYXR1cmFsIiwxXQ==",
      },
    },
    "tableSetQuery_sorted": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYW1lX2FzYyIsM10=",
          "node": Object {
            "name": "Budd Deey",
          },
        },
        Object {
          "cursor": "WyJuYW1lX2FzYyIsNV0=",
          "node": Object {
            "name": "Joe Tucker",
          },
        },
        Object {
          "cursor": "WyJuYW1lX2FzYyIsMV0=",
          "node": Object {
            "name": "John Smith",
          },
        },
        Object {
          "cursor": "WyJuYW1lX2FzYyIsNF0=",
          "node": Object {
            "name": "Kathryn Ramirez",
          },
        },
        Object {
          "cursor": "WyJuYW1lX2FzYyIsMl0=",
          "node": Object {
            "name": "Sara Smith",
          },
        },
        Object {
          "cursor": "WyJuYW1lX2FzYyIsNl0=",
          "node": Object {
            "name": "Twenty Seventwo",
          },
        },
      ],
      "pageInfo": Object {
        "endCursor": "WyJuYW1lX2FzYyIsNl0=",
        "hasNextPage": false,
        "hasPreviousPage": false,
        "startCursor": "WyJuYW1lX2FzYyIsM10=",
      },
    },
    "typesQuery": false,
  },
}
`;
